susemanager-utils/susemanager-sls/salt/switch_to_bundle/mgr_switch_to_venv_minion.sls

```diff
@@ -0,0 +1,19 @@
+{%- set avoid_venv_salt_minion = salt['pillar.get']('custom_info:mgr_avoid_venv_salt_minion') %}
```

Comment:
why `custom_info:mgr_avoid_venv_salt_minion` here and not `mgr_avoid_venv_salt_minion`?

In our SUSE Manager documentation we mention `mgr_venv_salt_minion` as the pillar to control this: https://documentation.suse.com/suma/4.3/en/suse-manager/client-configuration/contact-methods-saltbundle.html

To define this pillar, customer should go to `/srv/pillar/`, define `top.sls` and define this pillar manually.

Reply:
Changed. I used `custom_info:mgr_avoid_venv_salt_minion` because I was thinking that setting the `custom_info:mgr_avoid_venv_salt_minion` from the UI was the proper way to avoid the automigration from an already existing Salt 3000 client.  But if the client was bootstrapped with `mgr_venv_salt_minion` it is reasonable to use this pillar to control also this, as you mention.